packages/web-components/docs/api-report.md

```diff
@@ -1,1734 +1,1734 @@
-
-// @public (undocumented)
-export const typeRampPlus3FontVariations: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus3LineHeight: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus4: CSSDirective;
-
-// @public (undocumented)
-export const typeRampPlus4FontSize: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus4FontVariations: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus4LineHeight: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus5: CSSDirective;
-
-// @public (undocumented)
-export const typeRampPlus5FontSize: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus5FontVariations: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus5LineHeight: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus6: CSSDirective;
-
-// @public (undocumented)
-export const typeRampPlus6FontSize: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus6FontVariations: CSSDesignToken<string>;
-
-// @public (undocumented)
-export const typeRampPlus6LineHeight: CSSDesignToken<string>;
-
-// Warnings were encountered during analysis:
-//
-// dist/dts/color/palette.d.ts:70:5 - (ae-forgotten-export) The symbol "create" needs to be exported by the entry point index.d.ts
-// dist/dts/color/palette.d.ts:71:5 - (ae-forgotten-export) The symbol "from" needs to be exported by the entry point index.d.ts
-// dist/dts/custom-elements.d.ts:52:5 - (ae-incompatible-release-tags) The symbol "fluentAnchor" is marked as @public, but its signature references "Anchor" which is marked as @internal
-// dist/dts/custom-elements.d.ts:54:5 - (ae-incompatible-release-tags) The symbol "fluentBadge" is marked as @public, but its signature references "Badge" which is marked as @internal
-// dist/dts/custom-elements.d.ts:57:5 - (ae-incompatible-release-tags) The symbol "fluentButton" is marked as @public, but its signature references "Button" which is marked as @internal
-// dist/dts/custom-elements.d.ts:96:5 - (ae-incompatible-release-tags) The symbol "fluentTextArea" is marked as @public, but its signature references "TextArea" which is marked as @internal
-// dist/dts/custom-elements.d.ts:97:5 - (ae-incompatible-release-tags) The symbol "fluentTextField" is marked as @public, but its signature references "TextField" which is marked as @internal
-// dist/dts/custom-elements.d.ts:98:5 - (ae-incompatible-release-tags) The symbol "fluentToolbar" is marked as @public, but its signature references "Toolbar" which is marked as @internal
-// dist/dts/custom-elements.d.ts:99:5 - (ae-incompatible-release-tags) The symbol "fluentTooltip" is marked as @public, but its signature references "Tooltip" which is marked as @internal
-
-// (No @packageDocumentation comment for this package)
-
-```
+## API Report File for "@fluentui/web-components"
```

Comment:
This change is unrelated to Sankey chart. Should be reverted

Reply:
I would like to, but this is a LF to CRLF change introduced via a `git add *.* --renormalize`

Reply:
This is an autogenerated file and should not be edited manually. Could you revert it.
![image](https://github.com/microsoft/fluentui/assets/98592573/597ad127-04c1-4806-80c5-9d098536692f)